code/modules/mob/living/carbon/human/life/handle_chemicals_in_body.dm

```diff
@@ -79,7 +79,7 @@
 	if(stat != 2)
 		var/reduce_nutrition_by = HUNGER_FACTOR
 		if(sleeping)
-			reduce_nutrition_by *= 0.1 //Reduce hunger factor by 90%
+			reduce_nutrition_by *= 0.75 //Reduce hunger factor by 25%
```

Comment:
Can't you just like generate the same amount of body heat while burning less nutriment to keep this effect